reports.py

```diff
@@ -242,6 +241,20 @@ def have_team_no_untriaged_no_priority(reporter, issues):
         printer=make_console_printer(show_teams=True))
 
 
+def stale_pull_requests(reporter, issues, days):
+    def predicate(issue):
+        return \
```

Comment:
Rather than \ lines, it is nicer to write
    return (is_open(issue)
                and is pull(...)
                ...

Reply:
Nice. Done.